pydoctor/templates/apidocs.css

```diff
@@ -1,287 +1,210 @@
-/* body { */
-/*     background: white url("http://twistedmatrix.com/images/TwistedMatrixLabs.png"); */
-/*     background-repeat: no-repeat; */
-/*     background-position: 100% 10%; */
-/* } */
+.navbar {
+    margin-bottom: 0px;
+}
+
+.page-header {
+    margin-top: 22px;
+}
+
+
+ul {
```

Comment:
isn't this going to affect <ul>s in docstrings too?  feels like it should be a _bit_ more qualified.